internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
 		include: createFlags.Include,
 	}, nil
 }
+
+func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
+	log := output.NewStdoutLogger(output.InfoLog)
+
+	log.Info(fmt.Sprintf("%s What name would you like to give this new account?", output.GoEmoji()))
+	name := output.NamePrompt()
+
+	networkName, selectedNetwork := output.CreateAccountNetworkPrompt()
+	// create new gateway based on chosen network
+	gw, err := gateway.NewGrpcGateway(selectedNetwork.Host)
+	if err != nil {
+		return nil, err
+	}
+	privateJsonFileName := ""
+	saveToGitIgnore := false
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		privateJsonFileName = fmt.Sprintf("%s.private.json", name)
+		log.Info(fmt.Sprintf("%s For security purposes, the private key generated for this account will be "+
+			"stored in separate file: %s", output.WarningEmoji(), privateJsonFileName))
+		saveToGitIgnore = output.AddToGitIgnorePrompt(privateJsonFileName)
+	}
+	log.Info(fmt.Sprintf("%s This command will perform the following:\n", output.WarningEmoji()))
+	log.Info(fmt.Sprintf("- Generate a new ECDSA P-256 public and private key pair\n"))
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		if saveToGitIgnore {
+			log.Info(fmt.Sprintf("- Save the private key to %s\n", privateJsonFileName))
+		}
+	}
+	log.Info(fmt.Sprintf("- Create a new account on %s and pair the public key with the new account\n", networkName))
+	log.Info(fmt.Sprintf("- Save the newly created account configuration to flow.json\n"))
+	output.NextStepPrompt()
```

Comment:
This prompt should be changed so it's not a selection prompt:
```
Use the arrow keys to navigate: ↓ ↑ → ←
? Press <ENTER> to continue:
  ▸ CONTINUE
```

it should look like:
```
Press <ENTER> to continue
```
This way there is not a lot of screen property taken for this simple action.

Reply:
addressed